__tests__/app.test.js

```diff
@@ -97,4 +96,24 @@ describe('5. get /api/users', () => {
             })
         })
     });
+});
+
+describe('6', () => {
+    test('returns 200 and updated article', () => {
+        return request(app)
+        .patch("/api/articles/1")
+        .send({ inc_votes : 55})
+        .expect(200)
+        .then(({body}) => {
+            expect(body.article).toEqual({
```

Comment:
You could use objectContaining rather than toEqual to make this test a bit more flexible later on... 👀